ferrilata-devilline/Models/Error.cs

```diff
@@ -0,0 +1,40 @@
+namespace ferrilata_devilline.Models
+{
+    public class Error
+    {
+        public string error { get; set; }
+
+        public Error(string error)
+        {
+            this.error = error;
+        }
+
+        protected bool Equals(Error other)
+        {
+            return string.Equals(error, other.error);
+        }
+
+        public override bool Equals(object obj)
```

Comment:
if this is for the comparation feature, i'd recommend just using anonymous objects and deserializing them to string each time. But let Suvi decide if this matters.

Reply:
Sure, let's ask Suvi which way is the best.